src/main/java/org/openstreetmap/atlas/geography/atlas/items/AtlasEntity.java

```diff
@@ -194,6 +196,34 @@ public String toDiffViewFriendlyString()
      */
     public abstract LocationIterableProperties toGeoJsonBuildingBlock();
 
+    public abstract JsonObject asGeoJsonFeature();
+
+    /**
+     * A method that creates a GeoJSON feature with properties from the tags. Note that it does not
+     * contain the geometry, and the subclasses should call this and then fill in the geometry from
+     * within their implementation of asGeoJsonFeature().
+     *
+     * @return A GeoJSON Feature with properties but no geometry.
+     */
+    protected JsonObject asGeoJsonFeatureWithPropertiesOnly()
```

Comment:
A geojson feature without geometry seems incomplete. I think getGeojsonProperties() would be cleaner.

Reply:
Duh, good point, will do. Then I can have a feature function in GeoJsonUtil that can take a geometries and a properties and make the feature.